applications/PfemSolidMechanicsApplication/python_scripts/gauss_point_utility.py

```diff
@@ -43,8 +43,8 @@ def ExecuteFinalizeSolutionStep(self):
 
         elems = self.model_part.GetElements(0)
         for elem in elems: 
-            cauchy_vector = elem.GetValuesOnIntegrationPoints( KratosMultiphysics.CAUCHY_STRESS_TENSOR, SomeProcessInfo)
-            epsi_vector = elem.GetValuesOnIntegrationPoints( KratosMultiphysics.GREEN_LAGRANGE_STRAIN_TENSOR, SomeProcessInfo)
+            cauchy_vector = elem.CalculateOnIntegrationPoints( KratosMultiphysics.CAUCHY_STRESS_TENSOR, SomeProcessInfo)
```

Comment:
@AFranci you need to be aware of this.

Reply:
Hi @RiccardoRossi, thank you for the advice, however I am not the owner of the PfemSolid and I don't even work with it. Maybe @josep-m-carbonell should be interested in it.